src/testsuite/vec_perf_f64.c

```diff
@@ -73,12 +74,14 @@ int timed_is_f64 (void)
 
 int timed_fpclassify_f64 (void)
 {
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
// Clang does not define vec_test_data_class and friends.